core/src/main/java/io/aiven/klaw/controller/TemplateMapController.java

```diff
@@ -531,6 +531,15 @@ public String syncBackTopics(
     return checkAuth("syncBackTopics.html", request, response, abstractAuthenticationToken);
   }
 
+  @RequestMapping(value = "/syncBackSchemas", method = RequestMethod.GET)
+  public String syncBackSchemas(
```

Comment:
Maybe "GET /sync/schema/from/database" ?

Here i am half torn between staying consistent and updating them to be more restful :)

Reply:
changing this path is changing servlet context, and not easy to traverse between pages. so i like to keep this as is